src/ImageSharp.Interactive/ImageSharp.Interactive.csproj

```diff
@@ -18,4 +18,9 @@
     <PackageReference Include="SixLabors.ImageSharp" Version="1.0.3" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Include="$(OutputPath)/SixLabors.ImageSharp.Interactive.dll" Pack="true" PackagePath="interactive-extensions/dotnet" />
```

Comment:
When the package is loaded .net interactive check for dll in that path to find extensions and load them. Otherwise is just loading the dll without invoking the load extension code.

Reply:
Cool! I really need to have a thorough read through the docs. 😆